openfl/component/envoy/envoy.py

```diff
@@ -84,9 +86,9 @@ def run(self):
             self.is_experiment_running = True
             try:
                 with ExperimentWorkspace(
-                        self.name + '_' + experiment_name,
-                        data_file_path,
-                        is_install_requirements=True
+                        experiment_name=self.name + '_' + experiment_name,
```

Comment:
Consider using f-strings for readability

`experiment_name=f'{self.name}_{experiment_name}'`